src/screen.rs

```diff
@@ -21,7 +21,7 @@ use std::fmt::Display;
 
 /// Monitor struct containing data about a monitor on the system
 ///
-/// Use Screen::get_monitors() to return a Vec<Monitor> of all the monitors on the system
+/// Use `Screen::get_monitors()` to return a `Vec<Monitor>` of all the monitors on the system
```

Comment:
To fix rustdoc's `invalid_html_tags` warning.
`<Vector>` is seen as an (invalid) HTML tags if not inside backquotes.